src/app/components/task-form/task-form.component.ts

```diff
@@ -10,11 +11,17 @@ import {AddTaskComponent} from '../add-task/add-task.component';
 export class TaskFormComponent implements OnInit {
   task: string;
 
-  constructor(public dialogRef: MatDialogRef<AddTaskComponent>) {}
+  constructor(public dialogRef: MatDialogRef<AddTaskComponent>, private snackBar: MatSnackBar) {}
 
   ngOnInit(): void {}
 
   onNoClick(): void {
     this.dialogRef.close();
   }
+
+  openSnackBar(): void {
```

Comment:
don't need any changes in form, the form doesn't know about the result

Reply:
Done